client/me/help/chat-business-concierge-notice/index.jsx

```diff
@@ -19,27 +17,26 @@ import { isBusinessPlanUser } from 'state/selectors';
 
 class ChatBusinessConciergeNotice extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
-		isBusinessPlanUser: PropTypes.bool.isRequired,
 		from: PropTypes.string.isRequired,
 		selectedSite: PropTypes.object.isRequired,
 		to: PropTypes.string.isRequired,
-	};
 
-	static defaultProps = {
-		translate: identity,
+		// Connected props
+		isBusinessPlanUser: PropTypes.bool.isRequired,
+		moment: PropTypes.func.isRequired,
```

Comment:
`moment` and `translate` aren't connected props, they're provided from the `localize` HoC. Should we remove them, or move them above, or perhaps add a separate comment for those? I've seen several variations of separate comments we're adding for that case in Calypso, including:

```
// from localize()
// from HOCs
// i18n function provided by localize()
// From `localize`
// via `localize`
```

The list is not exhaustive. Perhaps pick one and go with it? 😉

Reply:
As I added the comment, I actually though "Well _actually_…", and then added the comment.

I think it can be helpful to distinguish between props we expect to be passed in a component hierarchy vs props that we expect to be there because their connected, from a HOC, or whatever. I don't think having a header for each type of these "present" props adds much value which is why I just left the `connected` header.

I'll give it a think, may just drop the separation.